file-formats/drty/examples/ztest_simple_type.drty.acds

```diff
@@ -0,0 +1,3 @@
+@EndUserText.label : 'This is a test label simple type'
+@EndUserText.quickInfo : 'This is the quick info for the simple type'
+define type ztest_simple_type : abap.char( 10 ); 
```

Comment:
@albertmink just mentioned to me it would be great if we could rename the example object to `zaff_example_drty`.

Would this be possible?

Reply:
Sure not problem